salt/nginx/etc/nginx.conf

```diff
@@ -34,42 +33,12 @@ http {
 	include             /etc/nginx/mime.types;
 	default_type        application/octet-stream;
 
-	# Load modular configuration files from the /etc/nginx/conf.d directory.
-	# See http://nginx.org/en/docs/ngx_core_module.html#include
-	# for more information.
 	include /etc/nginx/conf.d/*.conf;
 
-	#server {
-	#    listen       80 default_server;
-	#    listen       [::]:80 default_server;
-	#    server_name  _;
-	#    root         /opt/socore/html;
-	#    index        index.html;
-
-			# Load configuration files for the default server block.
-			#include /etc/nginx/default.d/*.conf;
-
-	#    location / {
-	#    }
-
-	#    error_page 404 /404.html;
-	#        location = /40x.html {
-	#    }
-
-	#    error_page 500 502 503 504 /50x.html;
-	#        location = /50x.html {
-	#    }
-	#}
-	server {
-		listen 80 default_server;
-		server_name _;
-		return 301 https://$host$request_uri;
-	}
-
-{% if FLEET_MANAGER %}
+	{%- if fleet_manager %}
 	server {
 		listen       8090 ssl http2 default_server;
-		server_name  _;
+		server_name  {{ url_base }};
```

Comment:
@defensivedepth Does a fleet manager need to be configured to respond based on this value or will it always be IP?

Reply:
@rwwiv It will be `url_base`